src/ipyaladin/widget.py

```diff
@@ -389,6 +394,58 @@ def target(self, target: Union[str, SkyCoord]) -> None:
             }
         )
 
+    def add_hips(self, hips: str, **hips_options: any) -> None:
+        """Add a HiPS to the Aladin Lite widget.
+
+        Parameters
+        ----------
+        hips : str
+            The HiPS to add to the widget.
```

Comment:
Tell what the user should put here. A CDS ID or an url: https://github.com/cds-astro/aladin-lite/blob/master/src/js/A.js#L110C14-L111C137